### PR TITLE
fix(FF): it works

### PR DESCRIPTION
**Pull request by mderrier**
use rollup to inject polyfill (FF < 63 support + IE 11) 
remove sound (chronoquiz crash)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,47 +76,47 @@
...
+        "rollup-plugin-polyfill": "3.0.0",
         "rollup-plugin-postcss": "^3.1.8",
         "rollup-plugin-terser": "^7.0.2",
         "rollup-plugin-typescript2": "^0.29.0",
         "ts-jest": "^26.4.4",
         "typescript": "^4.0.5"
     },
     "dependencies": {
-        "@webcomponents/webcomponentsjs": "2.5.0"
+        "@webcomponents/webcomponentsjs": "2.2.10"
```
Comment (mderrier) [inline package.json:119]: FF 54 doesn't work with webcomponents polyfill > 2.2

File: rollup.config.js [hunk 2]
```diff
@@ -172,6 +173,9 @@ const ROLLUP = [
                     moduleDirectory: 'submodules/math-json/src',
                 },
             }),
+            polyfill([
+                '@webcomponents/webcomponentsjs/webcomponents-bundle.js',
```
Comment (mderrier) [inline rollup.config.js:177]: inject polyfill via rollup
polyfill injected in code doesn't work with multi-instance

File: src/public/mathfield-element.ts [hunk 3]
```diff
@@ -365,10 +365,15 @@ export class MathfieldElement extends HTMLElement implements Mathfield {
     document.body.appendChild(mfe);
     * ```
     */
+
     constructor(options?: Partial<MathfieldOptions>) {
         super();
 
+        if ((window as any).ShadyCSS) {
+            (window as any).ShadyCSS.prepareTemplate(MATHFIELD_TEMPLATE, 'host');
```
Comment (mderrier) [inline src/public/mathfield-element.ts]: shadowCSS old browser support

File: src/public/mathfield-element.ts [hunk 4]
```diff
@@ -383,18 +388,18 @@ export class MathfieldElement extends HTMLElement implements Mathfield {
         );
 
         // Inline options (as a JSON structure in the markup)
-        try {
-            const json = slot
-                .assignedElements()
```
Comment (mderrier) [inline src/public/mathfield-element.ts:388]: no polyfill found/work

File: src/public/mathfield-element.ts [hunk 5]
```diff
@@ -722,9 +727,10 @@ export class MathfieldElement extends HTMLElement implements Mathfield {
         if (!this.hasAttribute('role')) this.setAttribute('role', 'textbox');
         // this.setAttribute('aria-multiline', 'false');
         if (!this.hasAttribute('tabindex')) this.setAttribute('tabindex', '0');
+        (window as any).ShadyCSS && (window as any).ShadyCSS.styleElement(this);
 
         this.#mathfield = new MathfieldPrivate(
-            this.shadowRoot.querySelector(':host > div'),
```
Comment (mderrier) [inline src/public/mathfield-element.ts:727]: ":host" selector doesn't work on older browser

**Review by mpech — APPROVED** [review]

**Review by arncet — APPROVED** [review]

**Pull request merged**